### PR TITLE
Some improvements.

### PR DESCRIPTION
**Pull request by MrAlexDeluxe**

### CODE REVIEW

File: zeeguu/static/js/DIST-zeeguu_learner_graph_module.js [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+// function convert_day_of_week_from_sun_to_mon
+var convert_day_of_week_from_sun_to_mon = function (date) {
+    return (date.getDay() + 6) % 7;
+};
+
+function week_number(date) {
+    var result, week_count_in_displayed_interval;
+    var days_in_month = 31;
+
+    if ((year_format(date) - year + 1) == 0) {
```
Comment (MrAlexDeluxe) [inline zeeguu/static/js/DIST-zeeguu_learner_graph_module.js]: I don't see a need to do that. For me it is more convenient to use "==" .

**Comment by mircealungu** [comment]
Aiii iaa!

In fact in your case, you must use == since it automatically converts the
LHS which is a string to an int [1]. Am I right to say that the LHS is a
string? At least it seems to me that year_format must return a string.

Thus, the alternative you'd have to write would be:

  parseInt(year_format()) - year + 1 === 0

M.


[1] -
http://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons

File: zeeguu/static/js/DIST-zeeguu_learner_graph_module.js [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+    html = html + "</div>";
+
+    html = html + '<div class="col-xs-4 col-md-4" style="padding-left: 30px;"> <h4>Current streak</h4>';
+    var current_streak_res = current_streak();
+    html = html + "<h3>" + current_streak_res + day_or_days(current_streak_res) + "</h3>";
+    html = html + "</div>";
+
+    html = html + "</div>";
+
+    document.write(html);
```
Comment (mircealungu) [inline zeeguu/static/js/DIST-zeeguu_learner_graph_module.js]: look into this pls

File: zeeguu/static/js/DIST-zeeguu_learner_graph_module.js (hunk 1 above)
Comment (mircealungu) [inline zeeguu/static/js/DIST-zeeguu_learner_graph_module.js]: we discussed about this

**Pull request merged**